package.json

```diff
@@ -12,6 +12,7 @@
     "dev": "concurrently \"npm run start\" \"grunt clientDev\"",
     "postinstall": "bower install && grunt clientBuild",
     "test": "grunt test",
+    "jest": "jest",
```

Comment:
I get an error when I run `npm run jest`:

```
No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In /Users/jswartz/src/opentok-rtc
  70 files checked.
  testMatch: **/__tests__/**/*.[jt]s?(x), **/?(*.)+(spec|test).[tj]s?(x) - 0 matches
  testPathIgnorePatterns: /node_modules/ - 70 matches
  testRegex:  - 0 matches
Pattern:  - 0 matches
```

Reply:
The CONTRIBUTING.md file says that you run the tests by running `grunt precommit`. Please update that file with the info on how to run all tests. Also, mention what the different npm scripts (test, jest, regression-test, and karma-test) do.

Reply:
removed Jest as snapshot testing doesnt really help us